workers/ohsome_quality_analyst/indicators/ghs_pop_comparison_roads/indicator.py

```diff
@@ -197,7 +193,6 @@ async def get_zonal_stats_population(self, bpolys: dict) -> Record:
                 st_setsrid(public.ST_GeomFromGeoJSON($3), 4326)
              )
             """
-        polygon = json.dumps(bpolys["features"][0]["geometry"])  # Geometry only
-        data = (polygon, polygon, polygon)
+        data = tuple(map(str, [self.feature.geometry] * 3))
```

Comment:
See comment in `ghs_pop_comparison_buildings/indicator.py`.

Reply:
I implemented your last suggestion